scripts/check_total_coverage.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+coverage_dir=$(PWD)/coverage
+combined_coverage_file=$coverage_dir/combined.txt
+COVERAGE_THRESHOLD=15.0 # percentage threshold of code coverage required
```

Comment:
lets not do any threshold check

Reply:
i added this check to make sure that none of our future PR will decrease the coverage, to guarantee that coverage % will only go up.

Reply:
ok please remove the success line, it is confusing to have success with just 15%
also put the percent into a variable

Reply:
i just removed that percentage check for now.